### DISCUSSION TITLE
New project with only two collaborators thus far

### DISCUSSION THREAD

**Discussion by Forrestey** [community/community#68966] [Programming Help]
Hi everyone check out my project id love to have some more people interested and collaborating together on this so far it's just me Forrestey and my friend jadedbabe working on it but it's a cool fun "little" project haha so check out the read me the wiki or browse the code and send some comits ! Even if you're a beginner like us we stay very organized and set up outlines, roadmaps, file tree and even example code before we start working on part of the project. Send nice we're beginners too it really makes learning while doing easier! Thanks and look forward to working with you all!

https://github.com/Forrestey/Fresh

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Forrestey ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Forrestey** [reply]
Thank you!

**Comment by Abkeyev99** [comment]
Thanks so good :)

**Comment by OSEight** [comment]
I would like to help!

**Reply by Forrestey** [reply]
Nice bro we’ve actually had a lot of interest so we’re taking things private but we are still meeting and discussing things with potential co-founders who would then in turn own equity in the project should they be selected. If you’re still interested let me know and we can chat about it. Did you have a chance to catch up on the read me file before we took the repo private?

**Reply by OSEight** [reply]
It does not exist.

**Reply by OSEight** [reply]
Did you mean next.js?

**Reply by OSEight** [reply]
I would still like to help, but I can not find the repository.

**Comment by OSEight** [comment]
<!-- xd --!>

**Comment by ibtissammassa** [comment]
Hey there! It sounds like a fun and organized project you've got going on. I'd love to check it out and possibly collaborate, but it seems like I'm having trouble finding the repository at the provided link. If there's an updated link or any specific instructions to locate the project, please let me know
